lib/vsc/install/commontest.py

```diff
@@ -212,3 +215,27 @@ def test_prospector(self):
         optparse.HelpFormatter.expand_default = orig_expand_default
 
         self.assertFalse(failures, "prospector failures: %s" % pprint.pformat(failures))
+
+    def test_readme_renderer(self):
+        """
+        The README has to render properly.
+        """
+
+        # inspired by the check -rs command readme_renderer.integration.distutils
+
+        # existence of README is checked in shared_setup
```

Comment:
nevertheless, shouldn't this be put in a try except clause? It's not because it exists that you can read it.